src/IpScanner/Program.cs

```diff
@@ -5,17 +5,24 @@ namespace IpScanner;
 
 internal static class Program
 {
-    private static bool IsDisplayUnknownHost => false; // TODO: add ability for change from args
+    private static bool IsDisplayUnknownHost = true;
 
     private static Action<string?> Print { get; } = Console.WriteLine;
 
     static void Main(string[] args)
     {
         try
         {
-            if (args.Length != 2)
+            if (args.Length < 2)
                 throw new Exception("Need two arguments: <Start IP> <End IP>");
 
+            if (args.Length == 3)
+            {
+                if (args[2] != "-u")
```

Comment:
just simplify it:
```cs
IsDisplayUnknownHost = args[2] == "-u";
```

Reply:
> if I do this then the third parameter will be ended up as required.
look at check above
> so are you sure about this?
certainly

Reply:
`if (args.Length == 3) ` this line is required because args[2]  will return null exception I believe you know why.

Reply:
yes :) so what is your original question?

Reply:
so are you margining the pull request or not?🤨

Reply:
yes